src/Raven.Client/Http/RavenCommand.cs

```diff
@@ -145,8 +149,7 @@ public virtual async Task<ResponseDisposeHandling> ProcessResponse(JsonOperation
                 }
 
                 // We do not cache the stream response.
-                using (var uncompressedStream = await RequestExecutor.ReadAsStreamUncompressedAsync(response).ConfigureAwait(false))
```

Comment:
Where is the call to `ReadAsStreamUncompressedAsync` now?

Reply:
This is draft PR so I temperately removed it, since I noticed that `await response.Content.ReadAsStreamAsync()` return `GZipStream`, so it make me think that maybe we don't need that function any longer (but I need to dig into it).

Reply:
Maybe I wrong, but to me it seems that `ReadAsStreamUncompressedAsync` was relevant only if the server return zipped response without the client supporting it. 

Is it even possible now? and if yes, why it was relevant only for `Raw` and not for `Object` response type as well.

But let's tackle this in a different issue I guess.

Reply:
No, you are correct that the http handler should be in charge of this